### PR TITLE
Prettier: also in gulp build for blocks-react

### PR DESCRIPTION
**Pull request by MagdalenaHenke**
Making a separate PR just to keep PRs smaller, but this one is build on top of the other prettier branch.

Adding a prettify step in the `gulp build` process for blocks-react.

### CODE REVIEW

File: packages/blocks-react/gulpfile.js [hunk 1]
```diff
@@ -1,30 +1,35 @@
-const { series, src, dest } = require('gulp');
-const babel = require('gulp-babel');
-const uglify = require('gulp-uglify');
-const sourcemaps = require('gulp-sourcemaps');
-const replace = require('gulp-string-replace');
-const del = require('del');
+const { series, src, dest } = require("gulp");
```
Comment (MagdalenaHenke) [inline packages/blocks-react/gulpfile.js]: most of these changes are from prettifying this file itself.

File: packages/blocks-react/gulpfile.js [hunk 2]
```diff
@@ -1,30 +1,35 @@
...
 
 function clean(cb) {
-  return del(['dist'], cb);
+  return del(["dist"], cb);
+}
+
+function prettify() {
+  return src("./components/*/*.jsx")
+    .pipe(prettier())
+    .pipe(gulp.dest("./components"));
```
Comment (MagdalenaHenke) [inline packages/blocks-react/gulpfile.js]: The only real "new" thing. Prettifying everything in `/component/` in place.

File: packages/blocks-react/package.json [hunk 3]
```diff
@@ -36,6 +36,7 @@
     "del": "^4.1.1",
     "gulp": "^4.0.2",
     "gulp-babel": "^7.0.1",
+    "gulp-prettier": "^2.3.0",
```
Comment (MagdalenaHenke) [inline packages/blocks-react/package.json]: https://www.npmjs.com/package/gulp-prettier

**Comment by MagdalenaHenke** [comment]
@bleveque how do you like this?

File: packages/blocks-react/gulpfile.js [hunk 4]
```diff
@@ -1,30 +1,35 @@
...
-const sourcemaps = require('gulp-sourcemaps');
-const replace = require('gulp-string-replace');
-const del = require('del');
+const { series, src, dest } = require("gulp");
+const babel = require("gulp-babel");
+const uglify = require("gulp-uglify");
+const sourcemaps = require("gulp-sourcemaps");
+const replace = require("gulp-string-replace");
+const del = require("del");
+const gulp = require("gulp");
```
Comment (bleveque) [inline packages/blocks-react/gulpfile.js]: would it be easy to combine this line with the `require("gulp")` up above?

File: packages/blocks-react/gulpfile.js [hunk 5]
```diff
@@ -1,30 +1,35 @@
...
+const prettier = require("gulp-prettier");
 
 function clean(cb) {
-  return del(['dist'], cb);
+  return del(["dist"], cb);
+}
+
+function prettify() {
+  return src("./components/*/*.jsx")
```
Comment (bleveque) [inline packages/blocks-react/gulpfile.js]: just to double-check, does this descend recursively into directories? or would we need `./components/**/*.jsx`?

would it be worth adding `...../*.js` here, too? and eventually typescript extensions?

File: packages/blocks-react/gulpfile.js (hunk 4 above)
Comment (MagdalenaHenke) [reply packages/blocks-react/gulpfile.js]: Yes, yes, absolutely (actually, I can just use the `dest` from above)

File: packages/blocks-react/gulpfile.js (hunk 5 above)
Comment (MagdalenaHenke) [reply packages/blocks-react/gulpfile.js]: eagle eye. 👏

**Comment by MagdalenaHenke** [comment]
@bleveque - fixed the things you pointed out - THANK YOU for the good look. Since it's a small PR and it gets merged into another one, I'm just gonna go ahead and merge it.

**Pull request merged**